graphql-java-support/src/main/java/com/apollographql/federation/graphqljava/SchemaTransformer.java

```diff
@@ -138,7 +139,7 @@ private String sdl() {
         final SchemaPrinter.Options options = SchemaPrinter.Options.defaultOptions()
                 .includeScalarTypes(true)
                 .includeExtendedScalarTypes(true)
-                .includeSchemaDefintion(true)
+                .includeSchemaDefinition(true)
```

Comment:
Do these changes mean that this version would not work with GraphQL-Java 13 at all? (Probably OK to require upgrades but worth thinking about.)

Reply:
Yes, that's correct unfortunately.